CHANGELOG.md

```diff
@@ -34,7 +34,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Raise `MisconfigurationException` when `enable_progress_bar=False` and a progress bar instance has been passed in the callback list ([#10520](https://github.com/PyTorchLightning/pytorch-lightning/issues/10520))
 
 
--
+- Changed initialization of progress bar callback to avoid setting `_progress_bar_callback` attribute on Trainer ([#10553](https://github.com/PyTorchLightning/pytorch-lightning/pull/10553))
```

Comment:
Note that updating the CHANGELOG for this PR is optional:

>  Did you update the CHANGELOG? (not for typos, docs, test updates, or internal minor changes/refactorings)

It falls under "internal minor changes/refactorings")